scripts/biomedical/chembl/test-data/chembl_29_expected.csv

```diff
@@ -0,0 +1,6 @@
+chembl_id,canonical_smiles,standard_inchi,standard_inchi_key,dcid
```

Comment:
Text values are not indicated correctly to make sure that commas are not incorrectly parsed. Quotes are missing from around some text values.